src/components/Modal/Modal.stories.mdx

```diff
@@ -2,17 +2,7 @@ import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs/blocks";
 
 import Modal from "./Modal";
 
-<Meta
-  title="Modal"
-  component={Modal}
-  argTypes={{
-    children: {
-      control: {
-        disable: true,
```

Comment:
This is required as you can't edit the children in the demo. We should also disable `buttons` as they also can't be edited.

Reply:
Ahh ok, I wanted to give people the ability to edit it in the demo. I can see now that it uses some weird `mdx` component structure instead of real react heh.